pkg/cedar/intake/schemas/easi_system_intake.json

```diff
@@ -54,152 +54,336 @@
+          ]
         },
         "costIncrease": {
-          "type": "string"
+          "type": "string",
+          "description": "Is there a cost increase associated with this request",
+          "examples": [
+            "YES"
+          ]
         },
         "costIncreaseAmount": {
-          "type": "string"
+          "type": "string",
+          "description": "How much is the cost increase",
+          "examples": [
+            "Over two million dollars"
+          ]
         },
         "decidedAt": {
-          "type": "string"
+          "type": "string",
+          "description": "Timestamp of when decision was reached",
+          "examples": [
+            "2022-02-17 14:34:43.405934+00"
+          ]
         },
         "decisionNextSteps": {
-          "type": "string"
+          "type": "string",
+          "description": "Steps that the business owner should take after receiving their decision",
+          "examples": [
+            "Go get a contract"
+          ]
         },
         "eaCollaborator": {
-          "type": "string"
+          "type": "string",
+          "description": "N/A",
+          "examples": [
+            "N/A"
+          ]
         },
         "eaCollaboratorName": {
-          "type": "string"
+          "type": "string",
+          "description": "Enterprise Architecture (EA) Collaborator",
+          "examples": [
+            "John Doe"
+          ]
         },
         "eaSupportRequest": {
           "enum": [
             "",
             "false",
             "true"
           ],
-          "type": "string"
+          "type": "string",
+          "description": "Does the request need EA support",
+          "examples": [
+            "True"
+          ]
         },
         "existingContract": {
-          "type": "string"
+          "type": "string",
+          "description": "Is there an existing contract for this effort",
+          "examples": [
+            "HAVE_CONTRACT"
+          ]
         },
         "existingFunding": {
           "enum": [
             "",
             "false",
             "true"
           ],
-          "type": "string"
+          "type": "string",
+          "description": "Will this project be funded out of an existing source",
+          "examples": [
+            "True"
+          ]
         },
         "fundingNumber": {
-          "type": "string"
+          "type": "string",
+          "description": "six digit funding number",
+          "examples": [
+            "123456"
+          ]
         },
         "fundingSource": {
-          "type": "string"
+          "type": "string",
+          "description": "Source of funding",
+          "examples": [
+            "Prog Ops"
+          ]
         },
         "grbDate": {
-          "type": "string"
+          "type": "string",
+          "description": "Scheduled date for the Governance Review Board (GRB) meeting",
+          "examples": [
+            "2025-12-12 00:00:00+00"
```

Comment:
See above comment about timestamp formatting.

Reply:
Still outstanding.

Reply:
Yeah apologies I missed this, fixed